pkg/authorization/apis/authorization/conversion.go

```diff
@@ -241,3 +243,42 @@ func Convert_rbac_PolicyRules_To_authorization_PolicyRules(in []rbac.PolicyRule)
 	}
 	return rules
 }
+
+func copyMapExcept(in map[string]string, except string) map[string]string {
+	out := map[string]string{}
+	for k, v := range in {
+		if k != except {
+			out[k] = v
+		}
+	}
+	return out
+}
+
+func convert_Origin_Annotations_To_rbac_Annotations(in map[string]string) map[string]string {
+	// can't use const in pkg/oc/admin/policy because of import cycle
+	value, ok := in["openshift.io/reconcile-protect"]
+	if ok {
+		out := copyMapExcept(in, "openshift.io/reconcile-protect")
+		if value == "true" {
+			out[rbac.AutoUpdateAnnotationKey] = "false"
+		} else {
```

Comment:
Why?
If there is an explicit annotation why would you remove it when converted ?
It would cause tests to fail too.